pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
@janschulz : I agree.  @jreback , there is no need to generate pickles with the `levels` parameter.  We already have tests for them that pass in this PR with no issues.  I think this is ready to be merged then since the potential backwards compatibility issue isn't actually a problem.